gdk/wizard/WizardChecker.py

```diff
@@ -77,3 +83,70 @@ def is_valid_custom_build_command(self, input_value):
             return False
         except ValueError:
             return len(input_value) > 0
+
+    def check_build_options(self, input_value):
+        try:
+            # Convert the input string to a dictionary (object)
+            input_obj = json.loads(input_value)
+        except json.JSONDecodeError:
+            return False
+
+        if not isinstance(input_obj, dict):
+            return False
+
+        if len(input_obj) > 2:
+            return False
+
+        if "excludes" not in input_obj or "zip_name" not in input_obj:
+            return False
+
+        excludes = input_obj.get("excludes")
+        if (
+            not isinstance(excludes, list)
+            or len(excludes) < 1
```

Comment:
Nit: Customers can specify empty list for `excludes`.

Reply:
I saw that on the schema it says : 
"options": {
                                                "type": "object",
                                                "description": "configuration options for the zip build system",
                                                "properties": {
                                                    "excludes": {
                                                        "type": "array",
                                                        "description": "regex patterns of files to exclude",
                                                        "minItems": 1
                                                    },
                                                    "zip_name": {
                                                        "type": "string"
                                                    }
                                                }
                                            }

Reply:
so i thought the minitem was at least one

Reply:
Got it. You can change it to empty list here.